src/libcollections/string.rs

```diff
@@ -337,8 +390,8 @@ impl String {
     ///
     /// ```
     /// // 𝄞music
-    /// let mut v = &mut [0xD834, 0xDD1E, 0x006d, 0x0075,
-    ///                   0x0073, 0x0069, 0x0063];
+    /// let v = &mut [0xD834, 0xDD1E, 0x006d, 0x0075,
```

Comment:
I think in these examples the `mut` here can be dropped as well (may have just bee included by accident long ago)